### ISSUE TITLE
Cannot uninstall poetry

### ISSUE THREAD

**Issue by kking39** [python-poetry/poetry#8068]
I need to uninstall poetry, but I am not able to do so following any of the advice that I found online. First, I confirmed that I do have poetry installed:

`% which poetry
/Library/Frameworks/Python.framework/Versions/3.10/bin/poetry
% poetry --version`
`Poetry (version 1.5.1)`

However, when I try to uninstall, I get this message:

`% curl -sSL https://install.python-poetry.org | python3 - --uninstall`
`Poetry is not currently installed.`

So, just for curiosity, I tried to re-install, and I get an error (understandably, since it is already installed):

`% curl -sSL https://install.python-poetry.org | python3 -`
`Retrieving Poetry metadata
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/urllib/request.py", line 1348, in do_open
    h.request(req.get_method(), req.selector, req.data, headers,
...
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)
`

I'd be grateful for any help or ideas about what else to try! Thank you.

**Comment by dimbleby** [comment]
the poetry installer doesn't put poetry there, and so it won't remove it from there either.

You need to remember what you did to install poetry ,and then do the opposite of that.

**Comment by kking39** [comment]
I ended up manually deleting poetry from all of its locations (I had it installed in multiple places). 

Then I still got an error when I tried to install poetry: "`unable to get local issuer certificate`". Fortunately, that was easy to resolve by following the instructions in the top answer here: https://stackoverflow.com/questions/68275857/urllib-error-urlerror-urlopen-error-ssl-certificate-verify-failed-certifica.

And then I was able to run the standard command and install poetry successfully: 

`curl -sSL https://install.python-poetry.org | python3 -`
